cmd/state/exec3/state.go

```diff
@@ -251,7 +252,24 @@ func (rw *Worker) RunTxTaskNoLock(txTask *state.TxTask, isMining, skipPostEvalua
 
 		// End of block transaction in a block
 		syscall := func(contract libcommon.Address, data []byte) ([]byte, error) {
-			return core.SysCallContract(contract, data, rw.chainConfig, ibs, header, rw.engine, false /* constCall */, rw.hooks)
+			ret, logs, err := core.SysCallContract(contract, data, rw.chainConfig, ibs, header, rw.engine, false /* constCall */, rw.hooks)
+			if err != nil {
+				return nil, err
+			}
+
+			for _, lg := range logs {
```

Comment:
`RunTxTaskNoLock` doesn't write directly to `domain` because it intent to be used in many parallel workers (unordered txs execution, etc...).

Let's add this data as a field to `TxTask` object, and in func `ApplyState4` write this field to `domains`. FYI: we have field `txTask.Logs` which written inside `ApplyState4 -> ApplyLogsAndTraces4`